data/human/hails.txt

```diff
@@ -3441,6 +3802,8 @@ phrase "test dummy"
 		"Warning! Warning!"
 		"Warning! Warning! Alien spacecraft approaching!"
 		"We have normality. I repeat, we have normality. Anything you still can't cope with is therefore your own problem."
+		"Look at your precious human ship, for the last time!"
+		"What? No! These aren't just pre-recorded messages!"
```

Comment:
This joke would also be lost on the regular player (since the player would normally see this message without any of the other messages).